spec/lib/audit/moab_to_catalog_spec.rb

```diff
@@ -41,6 +41,20 @@
     end
   end
 
+  describe "#seed_from_disk_with_profiling" do
+    let(:m2c) { described_class.new }
+
+    it "logs what it's about to do, uses the profiler, and prints profiling results" do
```

Comment:
seems like the logging part got left off?  or i'm just missing it.  but if it did get left off, it'd be nice to either add it, or fix the test description.  logging would be nice, i'd be in favor of adding it, but also doesn't feel super-necessary, so happy to have it omitted if it doesn't feel worth the effort.